### DISCUSSION TITLE
Configuration MR targets default branch

### DISCUSSION THREAD

**Discussion by georgkrause** [renovatebot/renovate#15020] [Request Help] [answered]
Hello everyone,

I am trying to setup renovate for a new project, and it seems to be impossible to tell renovate to file the Merge Request on a self hosted gitlab against the right branch. There is develop and master, while master is the default branch. And I have `RENOVATE_BASE_BRANCHES=develop' configured, but renovate opens its configuration merge request against the master branch.

Its kind of easy to avoid this, by 

- changing the default branch
- manually change the merge request and rebase the branch

I still think this might be a bug and want to let you know to see if you can fix it.

Thanks for the great tool!

**Comment by rarkins** [comment, marked as answer]
The config must exist in default branch, so it's expected behaviour
